terraform/workspace-variables/sandbox.tfvars

```diff
@@ -0,0 +1,28 @@
+# PaaS
+paas_app_environment             = "sandbox"
+env_config                       = "sandbox"
+paas_space_name                  = "sandbox"
+paas_postgres_service_plan       = "small-ha-11"
+paas_redis_service_plan          = "tiny-ha-5_x"
+paas_app_start_timeout           = "180"
+paas_web_app_deployment_strategy = "blue-green-v2"
+paas_web_app_instances           = 2
+paas_web_app_memory              = 512
+paas_worker_app_instances        = 2
+paas_worker_app_memory           = 512
+paas_worker_app_stopped          = false
+
+statuscake_alerts = {
+
+  alert = {
+    website_name   = "register-sandbox"
+    website_url    = "https://sandbox.register-trainee-teachers.education.gov.uk/ping"
+    test_type      = "HTTP"
+    check_rate     = 60
+    contact_group  = [188603]
+    trigger_rate   = 0
+    custom_header  = "{\"Content-Type\": \"application/x-www-form-urlencoded\"}"
```

Comment:
Could you please simplify and use the defaults? Look at https://github.com/DFE-Digital/publish-teacher-training/pull/1513

Reply:
fixed..